features/330-serious-violence-reduction-order-3324/pnc-data.before.xml

```diff
@@ -0,0 +1,27 @@
+<CPE02>
```

Comment:
Also not sure we need this

Reply:
Updated the file with the response from PNC.